types/react-router/test/hooks.tsx

```diff
@@ -5,13 +5,17 @@ interface Params {
     id: string;
 }
 
-interface LocationState {
-    s: string;
+declare module 'history' {
+    namespace History {
+        interface LocationState {
+            s?: string;
```

Comment:
I had to make these augmentations optional because the augmentation will be used for all tests!